ubg/src/common/Navbar.js

```diff
@@ -2,31 +2,65 @@ import React from 'react';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import MenuIcon from '@material-ui/icons/Menu';
+import Button from '@material-ui/core/Button';
 import {makeStyles} from '@material-ui/core/styles';
 import IconButton from '@material-ui/core/IconButton';
 import Typography from '@material-ui/core/Typography';
 import AuthButtons from './AuthButtons';
+import SearchField from './SearchField';
+import {useHistory} from 'react-router-dom';
 
 const useStyles = makeStyles((theme) => ({
   menuButton: {
     marginRight: theme.spacing(2),
   },
-  title: {
+  titleButton: {
     flexGrow: 1,
+    justifyContent: 'left',
+    textTransform: 'none',
+  },
+  Button: {
+    textDecoration: 'none',
+    color: 'white',
   },
 }));
 
+/**
+ * @return {ReactElement} Navigation bar of website
+ */
 export default function Navbar() {
   const classes = useStyles();
+  const history = useHistory();
+
+  /**
+   * Go to the home page url
+   */
+  function homePage() {
+    history.push(`/`);
```

Comment:
I think you should just use single quotes, it seems that backticks are not needed in this case.